test/test_operations_problem.jl

```diff
@@ -151,18 +141,19 @@ end
     parameters = [true, false]
     ptdf = PTDF(sys)
     # These are the duals of interest for the test
-    dual_constraint =
-        [[:nodal_balance_active__Bus], [:CopperPlateBalance, :network_flow__Line]]
+    dual_constraint = [[NodalBalanceActiveConstraint], [CopperPlateBalanceConstraint]]
```

Comment:
@sourabhdalvi @jd-lara I'm now working with this code. Should this interface take a ConstraintKey instead of ConstraintType (or tuple that can be used to create a ConstraintKey)? Before this PR I had written some code like this:
```
dual_constraint = [[(NodalBalanceActiveConstraint, PSY.Bus)], [(CopperPlateBalanceConstraint, PSY.System]]
```

Reply:
Since the dual constraint type gets passed with the model we can build the key at the construction time. The only exception is the case of the network where the device portion can be a system or a bus depending on the constraint type we want de dual from. 

I think we can still pass the constraint type and fill the remaining portion after since the system applies only in the case of the CopperPlateBalanceConstraint and Bus in any other constraint type.